content/rs/clusters/maintenance-mode.md

```diff
@@ -87,6 +88,9 @@ To turn maintenance mode off after you finish server maintenance, run:
 ```sh
 rladmin node <node_id> maintenance_mode off
 ```
+{{<note>}}
+“Maintenance mode off” can finish successfully even though it did not restore the snapshot.
```

Comment:
Hi @lanceleonard I need to change this into something else according to information from https://redislabs.atlassian.net/browse/RED-68917
The new text should say: 

Note that running “Maintenance mode off” will not have an effect if the correlating snapshot is missing.